.github/workflows/spec_test_on_nuttx.yml

```diff
@@ -159,6 +159,10 @@ jobs:
           echo "config INTERPRETERS_WAMR_AOT_STACK_FRAME" >> apps/interpreters/wamr/Kconfig
           echo "\tbool \"Enable AOT stack frame\"" >> apps/interpreters/wamr/Kconfig
           echo "\tdefault n" >> apps/interpreters/wamr/Kconfig
+          echo "\n" >> apps/interpreters/wamr/Kconfig
+          echo "config INTERPRETERS_WAMR_TAIL_CALL" >> apps/interpreters/wamr/Kconfig
```

Comment:
I think remove the nuttx specific config is better

Reply:
No idea how to do it? If I remove these lines, CI reported somewhat like "aot load fail: tail call is not enabled"

Reply:
Get it, so it's an missing option on NuttX?

Reply:
Yes, and I added an option in `product-mini/platforms/nuttx/wamr.mk`.

Reply:
Thanks, I'll submit this to NuttX side.